ingestion/src/metadata/ingestion/source/database/bigquery/queries.py

```diff
@@ -40,3 +40,12 @@
 BIGQUERY_TEST_STATEMENT = textwrap.dedent(
     """SELECT query FROM `region-{region}`.INFORMATION_SCHEMA.JOBS_BY_PROJECT limit 1"""
 )
+
+
+BIGQUERY_SCHEMA_DESCRIPTION = textwrap.dedent(
+    """
+    SELECT option_value as schema_description FROM
+    INFORMATION_SCHEMA.SCHEMATA_OPTIONS where schema_name = '{schema_name}'
+    and option_name = 'description' and option_value is not null
+    """
```

Comment:
can we pass the project id as well here? it is possible right that we might have the same schema name in multiple projects?

I think this might help: https://cloud.google.com/bigquery/docs/information-schema-datasets-schemata-options#scope_and_syntax

Reply:
nice catch, thanks